client/src/pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import InputTextField from "../../components/InputTextField/InputTextField";
+import Button from "../../components/Button/Button";
+//import Logo from "../Logo/Logo";
+import "./LoginPage.css";
+import LogoImg from "../../assets/images/login_logo.png";
+import { faDivide } from '@fortawesome/free-solid-svg-icons';
+
+const LoginPage = (props) => {
+  
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  function handleSubmit() {
+    console.log(email, password);
+  }
```

Comment:
This should probably be passed in as a prop, so that `App` can control the logged in user